packages/compass-aggregations/src/modules/pipeline-builder/stage-editor.ts

```diff
@@ -352,6 +352,7 @@ export const runStage = (
       };
       // We are not handling cancelling, just supporting `aggregatePipeline` interface
       const { signal } = new AbortController();
+      console.log('aggregatePipeline via runStage', new Error().stack);
```

Comment:
Accidentally committed? :)

Btw, just in case you find this helpful: https://developer.mozilla.org/en-US/docs/Web/API/console/trace

Reply:
yeah